apps/rxjs/hold-to-save-btn/src/app/holdable.directive.ts

```diff
@@ -0,0 +1,62 @@
+import { Directive, HostListener, input, output } from '@angular/core';
+import { toSignal } from '@angular/core/rxjs-interop';
+import {
+  Subject,
+  interval,
+  map,
+  merge,
+  switchMap,
+  takeUntil,
+  takeWhile,
+} from 'rxjs';
+
+type Ms = number;
+
+const DEFAULT_HOLD_TIME: Ms = 2000;
```

Comment:
maybe it's better if we can configurate this from the directive?

Reply:
I allow user to configurate this by provide a value to DEFAULT_HOLD_TIME_TOKEN